### ISSUE TITLE
GR feedback

### ISSUE THREAD

**Issue by matyaskopp** [clarin-eric/ParlaMint#448]
### missing notes
- [ ] speaker note
- [ ] vocal/@type="shouting"
- [ ] initial comments

It seemed to me strange that you have only one type of note, so I have checked one document that corresponds to the following:
https://github.com/clarin-eric/ParlaMint/blob/6be4475f8a5e0e990823dcc5440d469092bec156/Data/ParlaMint-GR/ParlaMint-GR_2020-01-13-S1-commons.xml#L156-L167

source files:
- pdf: https://www.hellenicparliament.gr/UserFiles/a08fc2dd-61a9-4a83-b09a-09f4c564609d/es20200113_1.pdf
- docx: https://www.hellenicparliament.gr/UserFiles/a08fc2dd-61a9-4a83-b09a-09f4c564609d/%ce%9f%ce%97%ce%84%2013-1-2020.docx
![image](https://user-images.githubusercontent.com/5867995/201133232-49f47cbc-9207-41e6-8a95-0f8fc8c33278.png)

the beginning of the document is pruned too
https://github.com/clarin-eric/ParlaMint/blob/6be4475f8a5e0e990823dcc5440d469092bec156/Data/ParlaMint-GR/ParlaMint-GR_2020-01-13-S1-commons.xml#L108-L114
![image](https://user-images.githubusercontent.com/5867995/201135320-9da27f3f-478a-44e2-a36c-4052abf23731.png)

### meeting
- [ ] sitting/session info in the `meeting` elements

the information about `sitting`/`session` is missing in the document header
https://github.com/clarin-eric/ParlaMint/blob/6be4475f8a5e0e990823dcc5440d469092bec156/Data/ParlaMint-GR/ParlaMint-GR_2020-01-13-S1-commons.xml#L12-L14

You have the information in the title.

**Comment by DimitrisGk-iel** [comment]
As different people are involved, we have scheduled a team meeting to discuss this as well as the email exchange we had and we will let you know

**Comment by DimitrisGk-iel** [comment]
**“ΟΛΟΙ ΟΙ ΒΟΥΛΕΥΤΕΣ: Μάλιστα, μάλιστα.”** 
Could you please provide a little more information regarding your comment with the arrow vocal? 
This translates into English as “ALL THE MEMBERS OF PARLIAMENT”: AGREED, AGREED. 
For us this should be encoded as vocal | type="shouting" . As a matter of fact, we have more than a couple of thousand lines of similar types that we have encoded them as vocal|shouting. 
“ALL THE MEMBERS OF PARLIAMENT”: yes, yes 
“ALL THE MEMBERS OF PARLIAMENT”: no 
“One member from the New democracy party”: shame! 
Could you please let us know how you would expect us to encode these use cases? 

**“SPEAKER NOTE before utterances”** 
We didn’t include this kind of notes because we have included into the u tag the information about who is talking and her/his role. In case you think it is essential information we could also add this kind of notes. 

**“the beginning of the document is pruned”** 
During our preprocessing we focused mainly at the speeches. All this introductory information (i.e. first paragraph) has been considered metadata and parts of it have been added in the teiHeader section. If you think that this kind of information should be included we could try to include it, using note and head tags. 

**“the information about sitting/session is missing in the document header”** 
As we mentioned in our email exchange the information is available and it is easy to be included. 

**What I have noticed is that you use only 3 NE classes, i.e. MISC is missing. Is this intentional?** 
From the annotations output by the NER module, we keep only PER, ORG, GPE and LOC, and we map GPE to LOC. If we are to submit a new version of our corpus we can also rerun our NER module and include in the output MISC NEs. What are your thoughts on that? 

**We seem count the number of words differently, If you would wish to fix that at your end to, pls. let me know and I can give you the exact (XSLT) code** 
We do want to use the same word count as you, so please provide us with the xslt code you mentioned in the email exchange.  

**I guess you took #covid to be >2019-10-30 but I take it to be >2019-11-01** 
Indeed, we have set the 2019-10-31 to be the last reference date. We will use the same date as you >2019-11-01 (strictly greater).

**Comment by matyaskopp** [comment]
> **“ΟΛΟΙ ΟΙ ΒΟΥΛΕΥΤΕΣ: Μάλιστα, μάλιστα.”**
> Could you please provide a little more information regarding your comment with the arrow vocal?
> This translates into English as “ALL THE MEMBERS OF PARLIAMENT”: AGREED, AGREED.
> For us this should be encoded as vocal | type="shouting" . As a matter of fact, we have more than a couple of thousand lines of similar types that we have encoded them as vocal|shouting.
> “ALL THE MEMBERS OF PARLIAMENT”: yes, yes
> “ALL THE MEMBERS OF PARLIAMENT”: no
> “One member from the New democracy party”: shame!
> Could you please let us know how you would expect us to encode these use cases?

Sorry, **you have encoded it properly**. 
I was probably insane when I was writing this issue.

---
> **“SPEAKER NOTE before utterances”**
> We didn’t include this kind of notes because we have included into the u tag the information about who is talking and her/his role. In case you think it is essential information we could also add this kind of notes.

Yes, I think that speaker notes shouldn't be skipped:
- You made an assumption based on these notes that the referred person in `who` is the speaker. But it is still an assumption, so adding the source(the note) of this assumption makes complete sense.
  - why assumption? There can be name duplicity, or the note can be wrong, but if you include a note, you can at least prove that you did not fabricate the speaker attribution.
- I am not aware of any other corpus that does not encode these notes = being consistent with others

@TomazErjavec ?

---
> “the beginning of the document is pruned”
> During our preprocessing we focused mainly at the speeches. All this introductory information (i.e. first paragraph) has been considered metadata and parts of it have been added in the teiHeader section. If you think that this kind of information should be included we could try to include it, using note and head tags.

I can imagine that this can be complicated to fix too. 
There can be an easy fix solution by adding something like this at the beginning of every pruned document:
```XML
<gap reason="editorial">
        <desc xml:lang="en">The frontmatter of the document has been removed</desc>
</gap>
```
but I am not sure if I like it.

@TomazErjavec please your opinion on this


---
> **“the information about sitting/session is missing in the document header”**
> As we mentioned in our email exchange the information is available and it is easy to be included.

Great, please do it

---
> **What I have noticed is that you use only 3 NE classes, i.e. MISC is missing. Is this intentional?**
> From the annotations output by the NER module, we keep only PER, ORG, GPE and LOC, and we map GPE to LOC. If we are to submit a new version of our corpus we can also rerun our NER module and include in the output MISC NEs. What are your thoughts on that?

I don't mind not having MISC because there is no proper definition of what NEs should be included in MISC. The note meant (I guess) that you should be aware that you have 3 NE classes, and it is not a bug.


---
> **We seem count the number of words differently, If you would wish to fix that at your end to, pls. let me know and I can give you the exact (XSLT) code**
> We do want to use the same word count as you, so please provide us with the xslt code you mentioned in the email exchange.

@TomazErjavec, I am not sure which script you are using

---
> **I guess you took #covid to be >2019-10-30 but I take it to be >2019-11-01**
> Indeed, we have set the 2019-10-31 to be the last reference date. We will use the same date as you >2019-11-01 (strictly greater).

yes please use covid start date >=2019-11-01

**Comment by TomazErjavec** [comment]
> > **“SPEAKER NOTE before utterances”**
> > We didn’t include this kind of notes because we have included into the u tag the information about who is talking and her/his role. In case you think it is essential information we could also add this kind of notes.
> 
> Yes, I think that speaker notes shouldn't be skipped:
> 
>     * You made an assumption based on these notes that the referred person in `who` is the speaker. But it is still an assumption, so adding the source(the note) of this assumption makes complete sense.
>       
>       * why assumption? There can be name duplicity, or the note can be wrong, but if you include a note, you can at least prove that you did not fabricate the speaker attribution.
> 
>     * I am not aware of any other corpus that does not encode these notes = being consistent with others

I agree with @matyaskopp's arguments. On the other hand, I have admit I actually don't know if all the other corpora do include all the notes as there is no formal way to check that. It is also true that ParlaMint is focused on transcripts. So, I would say, it would be great if you could retain the notes, but it is not a deal-braker.

> > “the beginning of the document is pruned”
> > During our preprocessing we focused mainly at the speeches. All this introductory information (i.e. first paragraph) has been considered metadata and parts of it have been added in the teiHeader section. If you think that this kind of information should be included we could try to include it, using note and head tags.
> 
> I can imagine that this can be complicated to fix too. There can be an easy fix solution by adding something like this at the beginning of every pruned document:
> 
> ```
> <gap reason="editorial">
>         <desc xml:lang="en">The frontmatter of the document has been removed</desc>
> </gap>
> ```
> 
> but I am not sure if I like it.
> 
> @TomazErjavec please your opinion on this

I agree, gap is ok, and meant for such cases, cf. [Sec. on gaps](https://clarin-eric.github.io/ParlaMint/#sec-gaps)

> > **“the information about sitting/session is missing in the document header”**
> > As we mentioned in our email exchange the information is available and it is easy to be included.
> 
> Great, please do it
> 
> > **What I have noticed is that you use only 3 NE classes, i.e. MISC is missing. Is this intentional?**
> > From the annotations output by the NER module, we keep only PER, ORG, GPE and LOC, and we map GPE to LOC. If we are to submit a new version of our corpus we can also rerun our NER module and include in the output MISC NEs. What are your thoughts on that?
> 
> I don't mind not having MISC because there is no proper definition of what NEs should be included in MISC. The note meant (I guess) that you should be aware that you have 3 NE classes, and it is not a bug.

Agree!

> > **We seem count the number of words differently, If you would wish to fix that at your end to, pls. let me know and I can give you the exact (XSLT) code**
> > We do want to use the same word count as you, so please provide us with the xslt code you mentioned in the email exchange.
> 
> @TomazErjavec, I am not sure which script you are using

Currently (and I am aware that this has to be changed to add-common-content soon!) it is parlamint2final.xsl, specifically this line: https://github.com/clarin-eric/ParlaMint/blob/72a3e13ca3a2e2fddf8708f687ec6a8723cb77a1/Scripts/parlamint2final.xsl#L109
which is supposed to translate into "all surface words" (cf. [Sec. on syntactic words](https://clarin-eric.github.io/ParlaMint/#sec-ana-norm)). Although, now that I look at it, it actually doesn't cover the second i.e. 2-1 case, but I don't think we have that anyway. I hope.

> > **I guess you took #covid to be >2019-10-30 but I take it to be >2019-11-01**
> > Indeed, we have set the 2019-10-31 to be the last reference date. We will use the same date as you >2019-11-01 (strictly greater).
> 
> yes please use covid start date >=2019-11-01

Actually, it should be > 2019-11-01 i.e. strictly greater.

**Comment by TomazErjavec** [comment]
> > **“SPEAKER NOTE before utterances”**
> > We didn’t include this kind of notes because we have included into the u tag the information about who is talking and her/his role. In case you think it is essential information we could also add this kind of notes.
> 
> Yes, I think that speaker notes shouldn't be skipped:
> 
>     * You made an assumption based on these notes that the referred person in `who` is the speaker. But it is still an assumption, so adding the source(the note) of this assumption makes complete sense.
>       
>       * why assumption? There can be name duplicity, or the note can be wrong, but if you include a note, you can at least prove that you did not fabricate the speaker attribution.
> 
>     * I am not aware of any other corpus that does not encode these notes = being consistent with others

I agree with @matyaskopp's arguments. On the other hand, I have admit I actually don't know if all the other corpora do include all the notes as there is no formal way to check that. It is also true that ParlaMint is focused on transcripts. So, I would say, it would be great if you could retain the notes, but it is not a deal-braker.

> > “the beginning of the document is pruned”
> > During our preprocessing we focused mainly at the speeches. All this introductory information (i.e. first paragraph) has been considered metadata and parts of it have been added in the teiHeader section. If you think that this kind of information should be included we could try to include it, using note and head tags.
> 
> I can imagine that this can be complicated to fix too. There can be an easy fix solution by adding something like this at the beginning of every pruned document:
> 
> ```
> <gap reason="editorial">
>         <desc xml:lang="en">The frontmatter of the document has been removed</desc>
> </gap>
> ```
> 
> but I am not sure if I like it.
> 
> @TomazErjavec please your opinion on this

I agree, gap is ok, and meant for such cases, cf. [Sec. on gaps](https://clarin-eric.github.io/ParlaMint/#sec-gaps)

> > **“the information about sitting/session is missing in the document header”**
> > As we mentioned in our email exchange the information is available and it is easy to be included.
> 
> Great, please do it
> 
> > **What I have noticed is that you use only 3 NE classes, i.e. MISC is missing. Is this intentional?**
> > From the annotations output by the NER module, we keep only PER, ORG, GPE and LOC, and we map GPE to LOC. If we are to submit a new version of our corpus we can also rerun our NER module and include in the output MISC NEs. What are your thoughts on that?
> 
> I don't mind not having MISC because there is no proper definition of what NEs should be included in MISC. The note meant (I guess) that you should be aware that you have 3 NE classes, and it is not a bug.

Agree!

> > **We seem count the number of words differently, If you would wish to fix that at your end to, pls. let me know and I can give you the exact (XSLT) code**
> > We do want to use the same word count as you, so please provide us with the xslt code you mentioned in the email exchange.
> 
> @TomazErjavec, I am not sure which script you are using

Currently (and I am aware that this has to be changed to add-common-content soon!) it is parlamint2final.xsl, specifically this line: https://github.com/clarin-eric/ParlaMint/blob/72a3e13ca3a2e2fddf8708f687ec6a8723cb77a1/Scripts/parlamint2final.xsl#L109
which is supposed to translate into "all surface words" (cf. [Sec. on syntactic words](https://clarin-eric.github.io/ParlaMint/#sec-ana-norm)). Although, now that I look at it, it actually doesn't cover the second i.e. 2-1 case, but I don't think we have that anyway. I hope.

> > **I guess you took #covid to be >2019-10-30 but I take it to be >2019-11-01**
> > Indeed, we have set the 2019-10-31 to be the last reference date. We will use the same date as you >2019-11-01 (strictly greater).
> 
> yes please use covid start date >=2019-11-01

Actually, it should be > 2019-11-01 i.e. strictly greater. Now that I think about it, it is a bit silly.

**Comment by TomazErjavec** [comment]
@matyaskopp, sorry, you are quite right, I have no idea where I took the `> 2019-11-01`, I must have been halucinating, and twice at that...
